baseline.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+'''
+Extract baseline from observations.
+
+    Example call:
+    baseline.py ./baseline.npz path/to/SST1M01_20171030.002.fits.fz
+
+Usage:
+  baseline.py <baseline_file_path> <files>...
+
+Options:
+  -h --help     Show this screen.
+'''
+from digicampipe.calib.camera import filter, r0
+from digicampipe.io.event_stream import event_stream
+from digicampipe.io.save_adc import save_dark
+from digicampipe.io.save_bias_curve import save_bias_curve
+from cts_core.camera import Camera
+from digicampipe.utils import geometry
+import matplotlib.pyplot as plt
+import numpy as np
+from tqdm import tqdm
+import pkg_resources
+from os import path
+from docopt import docopt
+
+
+def main(baseline_file_path, files):
+    camera_config_file = pkg_resources.resource_filename(
+        'digicampipe',
+        path.join(
+            'tests',
+            'resources',
+            'camera_config.cfg'
+        )
+    )
+
+    thresholds = np.arange(0, 400, 10)
+    unwanted_patch = [306, 318, 330, 342, 200]
+    unwanted_cluster = [200]
```

Comment:
make an unwanted_pixel list + introduce it as input arguments then use it in `set_pixels_to_zero()`

Reply:
remove unwanted_patch, unwanted_cluster

Reply:
can you explain why?

Reply:
basically in dark_evaluation.py script you want to extract all info regarding : dark events 

1. baseline, baseline std
2. trigger rat per patch
3. rate scan

but for baseline.py I assume you want it to just give you this dark.npz for running the crab_pipeline.py ? 
in that case you don't need the trigger information (in the R0Camera Container). Therfore only pixel images are important. The user would mask some failing part of the camera with pixels.
Sorry if it is not clear... gotta go. Will explain better :)

Reply:
> but for baseline.py I assume you want it to just give you this dark.npz for running the crab_pipeline.py ?

Yes exactly. That seemed to be useful.

> in that case you don't need the trigger information (in the R0Camera Container). Therfore only pixel images are important. The user would mask some failing part of the camera with pixels.

Aha .. I think I found my mistake. So I assumed (wrongly) that this means:
```
    unwanted_patch = [306, 318, 330, 342, 200]
    unwanted_cluster = [200]
```
these 5 patches and that cluster ... they are somehow broken and we do not want to look at them.
I assumed all of these pixels have a problem, so it is good to remove these pixel from all analysis.

This assumption was apparently wrong.

So I have a question: Why are these patches unwanted, and why is that cluster unwanted?

Reply:
Was that just an example? To show that patches and clusters can be removed?

Reply:
> Was that just an example? To show that patches and clusters can be removed?

Yes basically cluster 200 or more precisely cluster_7 contains patches [306,318,330,342,200]  (N.B. there is not 7 patches because it is at the camera border.)

Right now `set_*_to_zeros()` is just, as the name suggests, setting zero values to the raw data (sort of getting rid of noisy components). 

We might want to keep in mind that sometimes we could do some kind of flagging on the status of the pixels/patches/clusters. As there could be different actions to take depending on the hardware issue. For instance you could have : pixel 0 noisy and patch 0 (containing pixel 0) so you could decide to do : 

1. 
```python 
noisy_pixel = 0
image[noisy_pixel] = 0
```
2.
```python 
noisy_pixel = 0
image[noisy_pixel.patch.pixels] = 0
```
3.
```python 
noisy_pixel = 0
image[noisy_pixel.patch.pixels] = 0
trigger_in[noisy_pixel.patch] = 0
```
4.
```python 
noisy_pixel = 0
image[noisy_pixel] = 0
trigger_in[noisy_pixel.patch] = 0
```

This depends on what is the hardware issue.